FabricExample/src/examples/Svg.tsx

```diff
@@ -133,22 +133,22 @@ class SvgNativeMethods extends Component {
   };
   alert = () => {
     console.log("PRESSED");
-    this.root.toDataURL(base64 => {
+    this.root?.toDataURL(base64 => {
       this.setState({
         base64,
       });
     });
 
-    console.log(this.circle.isPointInFill({x: 200, y: 100}));
-    console.log(this.circle.isPointInStroke({x: 200, y: 100}));
-    console.log(this.circle.getTotalLength());
-    console.log(this.circle.getPointAtLength(25));
-    console.log(this.circle.getBBox({fill: true}));
-    console.log(this.circle.getCTM());
-    console.log(this.circle.getScreenCTM());
+    console.log(this.circle?.isPointInFill({x: 200, y: 100}));
+    console.log(this.circle?.isPointInStroke({x: 200, y: 100}));
+    console.log(this.circle?.getTotalLength());
+    console.log(this.circle?.getPointAtLength(25));
+    console.log(this.circle?.getBBox({fill: true}));
+    console.log(this.circle?.getCTM());
+    console.log(this.circle?.getScreenCTM());
   };
-  root: any;
-  circle: any;
+  root: Svg | null;
+  circle: Circle | null;
 
   componentDidMount() {
       this.alert();
```

Comment:
@WoLewicki, should we remove this componentDidMount() method from this example? (Or, we could also comment this out, and document why it doesn't work).

To summarize our discord discussion (for posterity): With Fabric, componentDidMount executes when the shadow tree gets committed (i.e: Fabric won't actually execute componentDidMount **after** the views are mounted on to the screen). So, trying to call a method like this.circle.getCTM(), which performs lookup on the view hierarchy (i.e: calling view.getParent()) inside componentDidMount is a bad idea.